Tasks/MavenV3/utils.ts

```diff
@@ -0,0 +1,125 @@
+import * as path from 'path';
+import * as fs from 'fs';
+import * as tl from 'azure-pipelines-task-lib/task';
+import * as xml2js from 'xml2js';
+import * as shelljs from 'shelljs'
+
+export async function readFile(filePath: string, encoding: string) {
+
+    tl.debug('Reading file at path ' + filePath)
+
+    return new Promise<string>((resolve, reject) =>
+        fs.readFile(filePath, (err, buffer) => {
+            const fileData = buffer.toString(encoding)
+            resolve(fileData)
+        })
+    )
+}
+
+function fixEncoding(str: string) {
```

Comment:
Was this temporary ported from common package? If so - please describe it in comments and point to the relevant ticket to return this implementation back to the common package.

Reply:
Moved it to the separate file and added the jsdoc description - for now it's independent utility